src/uct/tcp/tcp_cm.c

```diff
@@ -148,6 +148,44 @@ static void uct_tcp_cm_trace_conn_pkt(const uct_tcp_ep_t *ep,
             uct_tcp_ep_get_cm_id(ep));
 }
 
+ucs_status_t uct_tcp_cm_send_event_pending_cb(uct_pending_req_t *self)
+{
+    uct_tcp_ep_pending_req_t *req =
+            ucs_derived_of(self, uct_tcp_ep_pending_req_t);
+    ucs_status_t status;
+
+    status = uct_tcp_cm_send_event(req->ep, req->cm.event, req->cm.log_error);
+    if (status == UCS_OK) {
```

Comment:
uct_tcp_cm_send_event can return error (like CONN_RESET) so need to abort in such case:
```
if (status == NO_RESOURCE) {
    return status;
}

assert(status != INPROGRESS)
ucs_free(req)
return OK;
```

Reply:
done